documentation/docs/guidedtour.md

```diff
@@ -83,22 +83,31 @@ Copy the sources of the application into your own Git repository. While we will
 
 1. Provide a name for your new item (for example, *My First Pipeline*) and select **Multibranch Pipeline**.
    <p align="center">
-   ![Create Pipeline Job](images/JenkinsNewItemPipeline-1.png "Jenkins New Item")
+   ![Create Pipeline Job](images/JenkinsNewItemPipeline.png "Jenkins New Item")
    </p>
+   **Note:** The ready-made Continuous Delivery pipelines of project "Piper" require to run as **Multibranch Pipeline**.
 
-1. For **SCM**, choose **Git**.
-
-1. For **Repository URL** in the **Repositories** section, enter the URL of your Git repository, for example `https://github.com/<your-org>/cloud-cf-helloworld-nodejs`. **Note:** If your repository is protected, you must provide your credentials in the **Credentials** section.
+1. For **Branch Sources**, choose **Add source**, and select the **Git** as source repository.
    <p align="center">
-   ![Create Pipeline Job](images/JenkinsNewItemPipeline-2.png "Jenkins New Item")
+   ![Create Pipeline Job](images/JenkinsNewItemPipeline-AddSource.png "Branch Sources - Add source")
    </p>
 
-1. For **Branch Specifier** in the **Branches to build** section, enter the branch name `*/1_REST_persist_in_Memory`.
+1. For **Project Repository** in the **Git** section, enter the URL of your Git repository, for example `https://github.com/<your-org>/cloud-cf-helloworld-nodejs`. **Note:** If your repository is protected, you must provide your credentials in **Credentials**.
 
-1. Choose **Save**.
+1. For **Discover branches**, choose **Add** and select **Filter by name (with wildcards)**.
```

Comment:
Klickt man auf **Filter by name (with wildcards)**? Dann würde ich "choose **Add** and **Filter by name (with wildcards)**" schreiben.